src/table/utils/handle-key-press.ts

```diff
@@ -156,11 +214,11 @@ export const bodyHandleKeyPress = ({
   paginationNeeded,
   totalsPosition,
   selectionsAPI,
-}) => {
+}: handleBodyProps) => {
```

Comment:
we have `bodyHandleKeyUp` in the handle-key-press file. So it is better to rename it to be `handleBodyKeyDownProps`.

Reply:
done